app/routes/events/view/sessions/list.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    let route = document.location.href.substring(document.location.href.lastIndexOf('/') + 1);
```

Comment:
And don't split it from the URL. Not a good practice. Directly use the dynamic segment value.